search/includes/classes/class-search.php

```diff
@@ -632,7 +633,7 @@ public function filter__ep_feature_active( $active, $feature_settings, $feature
 	 * constant should be set to `true`, which will enable query integration for all requests
 	 */
 	public static function is_query_integration_enabled() {
-		if ( isset( $_GET['es'] ) ) {
+		if ( isset( $_GET[ self::QUERY_INTEGRATION_FORCE_ENABLE_KEY ] ) ) {
```

Comment:
Just noting we also need to update the comment for this function.

Reply:
Updated